deploy/templates/nstemplatetiers/base/ns_stage.yaml

```diff
@@ -57,6 +57,49 @@ objects:
   subjects:
   - kind: User
     name: ${USERNAME}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: exec-pods
```

Comment:
What do we need this role for?
Having an view role makes sense. Plus edit role would make sense for some sre accounts too. But how would you use the exec-pod role?

Reply:
My idea is that crtadmin users have the lowest needed permissions to be able to investigate a problem in a user's namespace and/or check if the user is mining bitcoins. To be able to exec into user's pod (and check what user is running there) you have to have permissions to `create` the `pods/exec` resource - and this is exactly what the role is about.

Reply:
Just to be clear - by the "lowest needed permissions" I meant that I don't want to give `edit` permissions to crtadmins - view + exec in pods should be enough.

Reply:
Ok. We can add more permissions later if needed.